### PR TITLE
RUBY-2008 Allow user to specify key alt names when creating a data key

### PR DESCRIPTION
**Pull request by egiurleo**
This is the first PR of a few, with the purpose of making it possible to refer to encryption keys by alternate names.

### CODE REVIEW

File: lib/mongo/client_encryption.rb [hunk 1]
```diff
@@ -53,6 +53,8 @@ def initialize(key_vault_client, options = {})
     #     by a colon (e.g. "kms.us-east-1.amazonaws.com" or
     #     "kms.us-east-1.amazonaws.com:443"). An endpoint in any other format
     #     will not be properly parsed.
+    # @option [ Array ] :key_alt_names An optional array of strings specifying
```
Comment (p-mongo) [inline lib/mongo/client_encryption.rb]: ` @option options [ Array<String> ] `

Comment (p-mongo) [reply lib/mongo/client_encryption.rb]: There's also the "options" between @option and the type.

Comment (egiurleo) [reply lib/mongo/client_encryption.rb]: I've been declaring these incorrectly 😆Just fixed all of them.

Comment (p-mongo) [reply lib/mongo/client_encryption.rb]: :+1:

**Pull request merged**